pkg/cmd/eventlistener/describe.go

```diff
@@ -68,9 +68,9 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
  {{decorate "bullet" $v.Name }}
 {{ " " }}
 {{- end }}
- BINDING NAME	KIND	APIVERSION
+ BINDING REF	KIND	APIVERSION
```

Comment:
Binding can have both `name`,  `ref` as shown [here](https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/cel-eventlistener-interceptor.yaml)
with the current change if EventListener  have both `name` and `ref` as shown [here](https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/cel-eventlistener-interceptor.yaml) then describe doesn't work properly for `name`

Reply:
Yes, for that I have created https://github.com/tektoncd/cli/issues/1004 to support this.